src/xpcc/driver/storage/block_device_mirror_impl.hpp

```diff
@@ -0,0 +1,101 @@
+// coding: utf-8
+/* Copyright (c) 2018, Raphael Lehmann
+ * All Rights Reserved.
+ *
+ * The file is part of the xpcc library and is released under the 3-clause BSD
+ * license. See the file `LICENSE` for the full license governing this code.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef XPCC_BLOCK_DEVICE_MIRROR_HPP
+	#error	"Don't include this file directly, use 'block_device_mirror.hpp' instead!"
+#endif
+#include "block_device_mirror.hpp"
```

Comment:
nope.

Reply:
Why? This includes don't hurt and make QtCreators syntax highlighting and auto-completion work.

Reply:
Yeah, ok, that's a good enough reason.